### DISCUSSION TITLE
Are there any known working macOS Calendar applications compatible with xandikos?

### DISCUSSION THREAD

**Discussion by walkertraylor** [jelmer/xandikos#172] [Q&A]
I have prohibitive issues with the apps I have tried.  Do any apps work reliably enough to use xandikos on a Mac?

On Calendar.app I have a problem renaming or deleting calendars.  
- I delete the calendar and click View->Refresh calendars.
- Error appears: The request (CalDAVCalendarSplitDualTypeCalendarQueueableOperation) for account “Localhost” failed.
- Many duplicates of the calendar to be deleted appear in the calendar list for that account.

On BusyCal I also experience the read-only issue #165

**Comment by jelmer** [comment]
The read-only calendar events were also reported in https://github.com/jelmer/xandikos/issues/165

Unfortunately I don't have access to a Mac myself, so I can't directly debug this.

**Reply by walkertraylor** [reply]
Okay.  I'm going to make an issue about Calendar.app for awareness in case a dev with a Mac comes across it.
